sdcm/nemesis.py

```diff
@@ -4080,10 +4086,13 @@ def disrupt_grow_shrink_cluster(self):
         if not self.has_steady_run and sleep_time_between_ops:
             self.steady_state_latency()
             self.has_steady_run = True
-        self._grow_cluster(rack=None)
+        new_nodes = self._grow_cluster(rack=None)
+
+        # pass on the exact nodes only if we have specific types for them
+        new_nodes = new_nodes if self.tester.params.get('nemesis_grow_shrink_instance_type') else None
```

Comment:
maybe we should support 2 parameters 

'nemesis_grow_instance_type'
'nemesis_shrink_instance_type'

because the current grow_shrink logic works like

1) setup cluster, nodes: a,b,c
2) grow, nodes a,b,c,d,e,f
3) shrink, nodes d,e,f

In the end, we have only new nodes. but with hawing nemesis grow shrink instance_type we will be able to grow_shrink only exactly the same nodded without the ability to touch existing

Reply:
if one want a full specific sequence, he can build a specific nemesis, or a specific test case.

I don't want to overload much more on this poor nemesis.

Reply:
I agree that the config is fine as is, we cannot make it to granular just to cover one test scenario that is probably not that interesting for the long run.

Reply:
but you will change the current logic of the test/menesis 

instead of having only new nodes at the end of the test, it will be old(initial) ones

Reply:
and that seems to be correct, assuming we design the test for specific node size, leaving cluster after this nemesis with smaller nodes will cause failures due to overload.